composition/package.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
-<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
```

Comment:
do we expect this to exist yet? (my linter's unhappy)

Reply:
no it doesnt exist yet. It will when we ratify the rep, in the meantime xmllint will complain but it won't impact any of our tools (https://github.com/ros2/rmw_implementation/pull/32#discussion_r148157220)